model_analyzer/triton/client/grpc_client.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
remove `(c)`

Reply:
I think it's fine to leave the (c) in to follow guidance from company, as long as this fits on one line with the auto-formatter, which it does.

Reply:
The correct string is `Copyright 20xx-20yy, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` So Tanmay is correct here.